src/Data.js

```diff
@@ -196,6 +196,7 @@ CM.ConfigData.DragonAuraInfo = {type: 'bool', group: 'Tooltip', label: ['Extra D
 // Statistics
 CM.ConfigData.Stats = {type: 'bool', group: 'Statistics', label: ['Statistics OFF', 'Statistics ON'], desc: 'Extra Cookie Monster statistics!', toggle: true};
 CM.ConfigData.MissingUpgrades = {type: 'bool', group: 'Statistics', label: ['Missing Upgrades OFF', 'Missing Upgrades ON'], desc: 'Shows Missing upgrades in Stats Menu. This feature can be laggy for users with a low amount of unlocked achievements.', toggle: true};
+CM.ConfigData.MissingAchievements = {type: 'bool', group: 'Statistics', label: ['Show Normal Missing Achievements', 'Show Shadow Missing Achievements', 'Show All Missing Achievements', 'Hide Missing Achievements'], desc: 'Shows Missing normal or shadow achievements in Stats Menu.', toggle: false};
```

Comment:
There is currently no way to turn of the missing achievements. I suggest giving the following options `Missing Achievements OFF`, `Missing Normal Achievements ON`, `Missing Shadow Achievements ON`, `Missing All Achievements ON`.

Reply:
I made it like the Scale one now and you can turn off it with the "Hide All Missing Achievements".
Tested after combining, works for me, the combined file is here: http://pneujai.ddns.net/src/js/CookieMonster.js

Reply:
Could you 1) merge the current dev-branch into your own dev-branch to apply all fixes that are now included and then 2) combine and push that combined file to your own dev-branch? That way I can easily check this PR without a whole lot of error messages related to other problems coming up.